sdk/src/client.ts

```diff
@@ -1,79 +1,32 @@
-import { WebSocket } from 'ws';
-import type { AuthStorage } from './composables/authentication.js';
-import { withoutTrailingSlash } from './utils.js';
-// import type { ReadItemsInput, ReadItemsOutput } from './commands/read-items.js';
-// import type { Command } from './types/index.js';
-
 export interface ClientConfig {
-	fetch?: fetch;
-	ws?: typeof WebSocket;
-	url?: string;
 	token?: string;
 }
 
-export interface BaseClientConfig extends Record<string, any> {
-	fetch: fetch;
-	ws: typeof WebSocket;
-	apiURL: string;
-}
-
-export interface GenericClient<Schema extends object, Features extends object> {
-	config: BaseClientConfig;
-	auth: AuthStorage;
```

Comment:
I believe I was able to "just" use the `DirectusClient` as the base client. From what I can tell from my tests everything still works as expected in schema/input/output types